packages/rum/package.json

```diff
@@ -17,7 +17,7 @@
     "build": "webpack",
     "build:dev": "webpack -w",
     "build:main": "npx babel src -d dist/lib --config-file ../../babel.node.js",
-    "build:e2e": "npm run script buildE2eBundles",
+    "build:e2e": "node ../../dev-utils/run-script.js buildE2eBundles packages/rum/test/e2e",
```

Comment:
nit: remove the `.js` extension from the scripts.

Reply:
Not sure about removing the extension, might be more clear to have it there. Don't have a strong opinion here though.

Reply:
Just to keep it consistent with other scripts in package.json and also feels easy to read on smaller widths(laptop screen).

Reply:
I think we eventually need to move everything under `rum/run-script.js` to `dev-utils/run-script.js` then we can use `npm run script` so let's clean this up in the next iteration when we remove `rum/run-script.js`